.github/workflows/js/check-jobs-for-errors.js

```diff
@@ -0,0 +1,18 @@
+module.exports = ({ core, jobs }) => {
```

Comment:
I might move this to a TS file under ci to make it consistent with version check on another PR

Reply:
Or maybe not as it would require a prepare job to run :(